src/app/_components/Search/Search.tsx

```diff
@@ -497,42 +535,91 @@ function SearchInput({ searchTermFromQueryParams = '' }) {
   );
 }
 
-export function Search() {
+export function Search({ fullScreen = false }: { fullScreen?: boolean }) {
+  const dispatch = useAppDispatch();
   const searchTermFromQueryParams = buildAdvancedSearchQuery(useFilterParams());
   const recentResults = useRecentResultsLocalStorage();
 
   const isSearchFieldFocused = useAppSelector(selectIsSearchFieldFocused);
   const searchTerm = useAppSelector(selectSearchTerm);
   const searchResponse = useSearchQuery(searchTerm, true);
   const isLoading = searchResponse.isLoading;
+  const isMobile = useBreakpointValue({ base: true, lg: false });
+
+  useEffect(() => {
+    if (isMobile && isSearchFieldFocused) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'auto';
+    }
+  }, [isMobile, isSearchFieldFocused]);
+
+  const Wrapper = useCallback(
+    function Wrapper({ children }: { children: ReactNode }) {
+      if (fullScreen) {
+        return (
+          <Flex
```

Comment:
Perhaps we should use the same implementation that we are using for the mobile menu, ie
    <Stack
      position="fixed"
      height="full"
      width="full"
      backgroundColor="surfaceTertiary"
      top={0}
      left={0}
      zIndex={'overlay'}
      gap={4}
      padding={6}
      overflow="hidden"
    > I think this would fix the issue where some ui elements are still showing up on top of the searchui